release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -521,6 +521,70 @@ If the following conditions are true, the feature is enabled by default:
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* The `NodeClockNotSynchronising` and `NodeClockSkewDetected` alerting rules are now disabled when the Precision Time Protocol (PTP) is in use.
+
+[id="ocp-4-15-monitoring-new-metrics-server-to-access-metrics-API-technology-preview"]
+==== New Metrics Server component to access the Metrics API (Technology Preview)
+This release introduces a Technology Preview option to add a Metrics Server component to the in-cluster monitoring stack.
+As a Technology Preview feature, Metrics Server is automatically installed instead of Prometheus Adapter if the `FeatureGate` custom resource is configured with the `TechPreviewNoUpgrade` option.
+If installed, Metrics Server, rather than Prometheus Adapter, collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs.
+Using Metrics Server instead of Prometheus Adapter frees the core platform Prometheus stack from handling this functionality.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#metricsserverconfig[MetricsServerConfig] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-new-feature-to-send-exemplar-data-to-remote-write-storage-for-user-defined-projects"]
+==== New feature to send exemplar data to remote write storage for user-defined projects
+User-defined projects can now use remote write to send exemplar data scraped by Prometheus to remote storage.
+To use this feature, configure remote write using the `sendExemplars` option in the `RemoteWriteSpec` resource.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#remotewritespec[RemoteWriteSpec] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-impoved-alert-querying-for-user-defined-projects"]
+==== Improved alert querying for user-defined projects
+Applications in user-defined projects now have API access to query alerts for application namespaces via the rules tenancy port for Thanos Querier. 
+You can now construct queries that access the `/api/v1/alerts` endpoint via port 9093 for Thanos Querier, provided that the HTTP request contains a `namespace` parameter.
+In previous releases, the rules tenancy port for Thanos Querier did not provide API access to the `/api/v1/alerts` endpoint.
+
+[id="ocp-4-15-monitoring-prometheus-updated-to-tolerate-jitters-at-scrape-time"]
+==== Prometheus updated to tolerate jitters at scrape time
+The default Prometheus configuration in the monitoring stack has been updated so that jitters are tolerated at scrape time.
+For monitoring deployments that have showed sub-optimal chunk compression for data storage, this update helps to optimize data compression, thereby reducing the disk space used by the time series database in these deployments.
```

Comment:
```suggestion
For monitoring deployments that have shown sub-optimal chunk compression for data storage, this update helps to optimize data compression, thereby reducing the disk space used by the time series database in these deployments.
```

Reply:
'shown` is the correct form with the particple: https://www.thesaurus.com/e/grammar/showed-or-shown/